packages/components/src/calendar/stories/date-calendar.story.tsx

```diff
@@ -112,7 +112,12 @@ export default meta;
 
 type Story = StoryObj< typeof DateCalendar >;
 
-export const Default: Story = {};
+export const Default: Story = {
+	args: {
+		defaultSelected: new Date( 1751472000000 ),
+		selected: new Date( 1751558400000 ),
+	},
+};
```

Comment:
Fixed TS errors. I'm not sure why these TS errors weren't caught in the previous PR. 


https://github.com/WordPress/gutenberg/blob/9504e4077d0991dc72bd27859725c011b1b42eba/packages/components/src/calendar/types.ts#L310-L318

Reply:
You're right! Previously, I encountered TS errors that stopped me from committing the changes. But now, I don't see any errors. I'll undo the changes.

Reply:
Reverted e66276b26e0b85bd7160f9259b1c58ab3a2af14e